src/python/WMComponent/DBS3Buffer/MySQL/CheckStatusForCompletedWorkflows.py

```diff
@@ -0,0 +1,57 @@
+from __future__ import print_function, division
+from WMCore.Database.DBFormatter import DBFormatter
+
+class CheckStatusForCompletedWorkflows(DBFormatter):
+    """
+    _CheckStatusForCompletedWorkflow_
+
+    Retrieve information about the workflow which checks the status of PhEDEx and DBS injection.
+    There is case task is completed but no file is associated to it. (EmptyBlock flag) 
+    Not sure how that happens
+    """
+    sql = """SELECT dw.name, df.in_phedex, df.status, count(*) as count
+                FROM dbsbuffer_workflow dw  
+                INNER JOIN wmbs_workflow ww on ww.name = dw.name
+                LEFT JOIN dbsbuffer_file df on dw.id = df.workflow
+             WHERE dw.completed = 1 group by dw.name, df.in_phedex, df.status """
+
+    def format(self, results):
+        results = self.formatDict(results)
+        resultByWorkflow = {}
+        for info in results:
+            workflow = info['name']
+            resultByWorkflow.setdefault(workflow, {})
+            resultByWorkflow[workflow].setdefault("InDBS", 0)
+            resultByWorkflow[workflow].setdefault("NotInDBS", 0)
+            resultByWorkflow[workflow].setdefault("InPhEDEx", 0)
+            resultByWorkflow[workflow].setdefault("NotInPhEDEx", 0)
+            resultByWorkflow[workflow].setdefault("EmptyTasks", 0)
+            
+            count = info["count"]
+            
+            if info['status'] == "InDBS":
```

Comment:
GLOBAL seems to be another popular state. This is something I want to investigate first to see whether we really need these 3 DBS status and what exactly is their meaning.